react/javascript/.rsync

```diff
@@ -1,3 +1,3 @@
 ../../LICENSE LICENSE
 ../../.templates/github/ .github/
-../../.templates/javascript/ .
+../../.templates/javascript/ . --exclude .mocharc.json --exclude tsconfig.json
```

Comment:
Not ideal to opt out of the sync here, but I do think `react/javascript` is a bit of a special case?

Reply:
Neat trick!